earn/src/components/portfolio/modal/WithdrawModal.tsx

```diff
@@ -271,7 +277,7 @@ export default function WithdrawModal(props: WithdrawModalProps) {
               size='L'
               onClick={() => {
                 if (maxWithdraw) {
-                  setInputValue(maxWithdrawBalance);
+                  setInputValue(bigMaxWithdraw.div(10 ** selectedOption.decimals).toFixed());
```

Comment:
```suggestion
                  setInputValue(bigMaxWithdraw.div(10 ** selectedOption.decimals).toFixed(selectedOption.decimals));
```
We should always be explicit with this.